contentcuration/contentcuration/views/internal.py

```diff
@@ -63,14 +62,14 @@ def handle_server_error(request):
 def authenticate_user_internal(request):
     """ Verify user is valid """
     logging.debug("Logging in user")
-    return HttpResponse(json.dumps({
+    return Response({
         'success': True,
         'user_id': request.user.id,
         'username': unicode(request.user),
         'first_name': request.user.first_name,
         'last_name': request.user.last_name,
         'is_admin': request.user.is_admin,
-    }))
+    })
```

Comment:
Woo hoo this works. I wanted to do that yesterday, but I didn't thinking ricecooker will break since we're changing to json content type will break something. Nope everything works fine (see [calling code here](https://github.com/learningequality/ricecooker/blob/master/ricecooker/commands.py#L190-L192))

The whole chef ran fine, so I assume all the other changes are fine too (at least for the ricecooker endpoitns). Will check in more detail tomorrow.